dependencies/README.md

```diff
@@ -5,6 +5,15 @@ Linkage Checker is a tool that finds [linkage errors](
 path. It scans the class files in the class path for references to other
 classes and reports any reference that cannot be satisfied.
 
+#### User Documentation
+
+We provide Linkage Checker with integrations for Maven and Gradle. Here are the user documentation:
```

Comment:
Linkage Checker can be used from Maven or Gradle.

Reply:
Updated.